src/System.Runtime.Extensions/tests/System/Convert.netcoreapp.cs

```diff
@@ -157,5 +148,29 @@ public void TryFromBase64Chars_MatchesFromBase64CharArray(string stringInput)
             Assert.Equal<byte>(expected, dest.Slice(0, dest.Length - 1).ToArray());
             Assert.Equal(0, dest[dest.Length - 1]);
         }
+
+        public static IEnumerable<object[]> FromBase64TestData
+        {
+            get
+            {
+                yield return new object[] { "" };
+                yield return new object[] { " \r\n\t" };
+                yield return new object[] { "BQYHCA==" };
+                yield return new object[] { " BQYHCA==" };
+                yield return new object[] { "B Q Y H C A = =" };
+                yield return new object[] { "B Q Y H C A = =" };
+                yield return new object[] { "BQYHCA==   " };
```

Comment:
Are leading white-spaces allowed? If so, we should permutations of those as well.